src/main/java/htsjdk/samtools/SamReaderFactory.java

```diff
@@ -142,6 +142,9 @@ public SamReader open(final Path path,
     /** Utility method to open the file get the header and close the file */
     abstract public SAMFileHeader getFileHeader(File samFile);
 
+    /** Utility method to open the file get the header and close the file */
+    abstract public SAMFileHeader getFileHeader(Path samFile);
```

Comment:
Maybe HTSJDK should start deprecating the `File` versions of the methods in favor of the `Path` versions. Anyway, for users using `File` the change is as simple as using `File.toPath` if there are no differences in the methods...

Reply:
I think we should probably only start doing that when we have full support of Path....